notebooks/official/pytorch/pytorch-text-sentiment-classification-custom-train-deploy.ipynb

```diff
@@ -0,0 +1,1979 @@
+{
```

Comment:
<p>Create a Dockerfile with TorchServe as the base image by performing the following steps:</p><p> </p><p>Add periods after all list items.</p><p> </p><p>Installs -&gt; Install</p><p> </p><p>to <code>/home/model-server/</code> directory -&gt; to the <code>/home/model-server/</code> directory</p><p> </p><p>of the container image -&gt; in the container image</p><p> </p><p>Add custom handler script -&gt; Add the custom handler script to the</p><p> </p><p>of the container image -&gt; in the container image</p><p> </p><p>the image <code>/home/model-server/</code>. -&gt; the <code>/home/model-server/</code> directory in the container image.</p><p> </p><p>the <code>/home/model-server/model-store/</code> -&gt; the <code>/home/model-server/model-store/</code> directory</p><p> </p><p>with name same as -&gt; with the name</p><p> </p><p>Torchserve -&gt; the TorchServe</p><p> </p><p>referencing -&gt; that references</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1166/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='76'/>